Documentation/files/multi-cluster/directory-sync-secret.yaml

```diff
@@ -0,0 +1,12 @@
+kind: Secret
```

Comment:
I think we should be able to remove this file since it's creation is documented.